Cargo.toml

```diff
@@ -24,3 +24,6 @@ members = [
     "pgx-examples/strings",
     "pgx-examples/triggers",
 ]
+
+[profile.dev.build-override]
+opt-level = 3
```

Comment:
TIL about https://doc.rust-lang.org/cargo/reference/profiles.html#build-dependencies, very neat.

Reply:
yeah, there.  So this means `build.rs` **and** its dependencies will always be compiled in release mode.  Which makes `build.rs` go about 2x faster, even if `build.rs` itself has to be recompiled because of a different `pgXX` feature flag -- its dependencies are still in release mode so they don't have to be recompiled.